source/VMelnalksnis.PaperlessDotNet/Documents/DocumentClient.cs

```diff
@@ -109,6 +110,23 @@ public async IAsyncEnumerable<Document<TFields>> GetAll<TFields>(int pageSize, [
 		return await GetCore<Document<TFields>>(id, cancellationToken).ConfigureAwait(false);
 	}
 
+
+	/// <inheritdoc />
+	public async Task<DocumentContent> Download(int id, bool original, CancellationToken cancellationToken = default)
+	{
+		var response = await _httpClient.GetAsync(original ? Routes.Documents.DownloadUri(id).ToString() :
+															 Routes.Documents.DownloadOriginalUri(id).ToString());
+
+		await response.EnsureSuccessStatusCodeAsync().ConfigureAwait(false);
+
+		var headers = response.Content.Headers;
+
+		return new DocumentContent(
```

Comment:
Can omit type because it is specified in method signature
```suggestion
		return new(
```